Modules/Core/Common/include/itkShapedImageNeighborhoodRange.h

```diff
@@ -388,9 +388,13 @@ class ShapedImageNeighborhoodRange final
      */
     QualifiedIterator() = default;
 
-    /** Constructor that allows implicit conversion from non-const to const
-     * iterator. Also serves as copy-constructor of a non-const iterator.  */
-    QualifiedIterator(const QualifiedIterator<false> & arg) noexcept
+    QualifiedIterator(const QualifiedIterator<VIsConst> & arg) noexcept = default;
+
+    /** Constructor that for implicit conversion from non-const to const
+     * iterator.  */
```

Comment:
This would introduce a typo in the documentation, obviously. You may simply remove `that` to fix the documentation.